Client/Frontend/Settings/HomepageSettings/HomePageSettingViewController.swift

```diff
@@ -122,7 +122,8 @@ class HomePageSettingViewController: SettingsTableViewController, FeatureFlaggab
 
         let pocketSponsoredSetting = BoolSetting(
             with: .sponsoredPocket,
-            titleText: NSAttributedString(string: .Settings.Homepage.CustomizeFirefoxHome.SponsoredPocket))
+            titleText: NSAttributedString(string: .Settings.Homepage.CustomizeFirefoxHome.ThoughtProvokingStories),
+            statusText: NSAttributedString(string: .Settings.Homepage.CustomizeFirefoxHome.PoweredByPocket))
```

Comment:
I made the suggestion in the other file, but not that one 🫣the build will fail
```suggestion
            statusText: NSAttributedString(string: .Settings.Homepage.CustomizeFirefoxHome.ThoughtProvokingStoriesSubtitle))
```